content/200-concepts/100-components/01-prisma-schema/04-data-model.mdx

```diff
@@ -819,6 +842,32 @@ You can optionally define a [custom unique constraint name](/concepts/components
 
 </Admonition>
 
+By default, the name of this field in Prisma Client queries will be `authorId_title`.
+
+You can also provide your own name for the composite unique constraint using the [`@@unique`](/concepts/components/prisma-schema/names-in-underlying-database#constraint-and-index-names) <span class="api"></span> attribute's `name` field:
+
+```prisma highlight=10;normal
+model Post {
+  id         String     @id @default(auto()) @map("_id") @db.ObjectId
+  createdAt  DateTime   @default(now())
+  title      String
+  published  Boolean    @default(false)
+  author     User       @relation(fields: [authorId], references: [id])
+  authorId   String     @db.ObjectId
+  categories Category[] @relation(references: [id])
+
+  @@unique(name: "authorTitle", [authorId, title])
+}
+```
+
+The `authorId_title` field will now be named `authorTitle` instead.
+
+<Admonition type="info">
+
+Refer to the documentation on [working with composite unique identifiers](/concepts/components/prisma-client/working-with-fields/working-with-composite-ids) to learn how to interact with a composite unique constraints in Prisma Client.
```

Comment:
broken link, probably should link here: https://docs-git-compound-id-unique-id-section-prisma.vercel.app/concepts/components/prisma-client/working-with-fields/working-with-composite-ids-and-constraints